components/alert/alert.css

```diff
@@ -0,0 +1,51 @@
+.alert__demo {
+  flex-wrap: wrap;
+  gap: 10px;
+  padding: 2rem;
+}
+.alert {
+  align-content: center;
+  padding: 1rem;
+  border-radius: 0.5rem;
+  border: 0.1rem solid;
+  width: 40%;
+  min-width: 30rem;
+}
+.alert__copy {
+  font-size: 1.6rem;
+  font-family: var(--secondary-font-family);
+  height: fit-content;
+  padding-left: 1rem;
+  gap: 1rem;
+}
+.alert__title {
+  padding: 0.2rem 0;
+  
+}
+.alert--info {
+  background-color: var(--alert-secondarycolor-info);
+  color: var(--alert-primarycolor-info);
+  border-color: var(--alert-primarycolor-info);
+}
```

Comment:
maybe use a hyphen between like this primary-color

Reply:
the name for it was already pretty big so thought i could exclude the hyphen there .